ddraw/IDirectDrawX.cpp

```diff
@@ -2735,7 +2735,7 @@ HRESULT m_IDirectDrawX::CreateD3D9Device()
 		// Auto stencel format
 		presParams.AutoDepthStencilFormat = (Config.DdrawOverrideStencilFormat) ? (D3DFORMAT)Config.DdrawOverrideStencilFormat : DepthStencilSurface;
 		// Auto stencel
-		presParams.EnableAutoDepthStencil = (presParams.AutoDepthStencilFormat != D3DFMT_UNKNOWN) ? TRUE : FALSE;
+		presParams.EnableAutoDepthStencil = (Config.DdrawOverrideStencilFormat || presParams.AutoDepthStencilFormat != D3DFMT_UNKNOWN) ? TRUE : FALSE;
```

Comment:
There should be no need to check Config.DdrawOverrideStencilFormat here as it is already handled in the line above.  As long as the line above assigns a value to `presParams.AutoDepthStencilFormat` then it will enable.